charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: {{ include "priorityclassversion" . }}
 kind: PriorityClass
 metadata:
   name: fluent-bit
-value: 50
+value: 101
```

Comment:
What do you think to set it to 120 or 150, just to have some free pod priorities for future use between fluent-bit and gardener-shoot-controlplane?

Reply:
Good point. Thanks